docs/routing/methods.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Methods and Events
+name: methods
+parent: Routing and Navigation
+nav_order: 4
+---
+
+## Methods
+
+Navigation can be triggered by method the [navTo](https://sapui5.hana.ondemand.com/#/api/sap.ui.core.routing.Router%23methods/navTo){:target="_blank"} on `sap.ui.core.routing.Router`.
```

Comment:
the method

Reply:
also, try using `https://ui5.sap.com` as the documentation domain - b/c reasons, you know ... :)